packages/video-feature/style.module.css

```diff
@@ -0,0 +1,43 @@
+.videoFeature {
+  --padding-y: 48px;
+  --padding-x: 48px;
+  --background-color: var(--wpl-neutral-100);
+
+  padding: var(--padding-y) var(--padding-x);
+  background-color: var(--background-color);
+  display: flex;
+  flex-direction: column-reverse;
```

Comment:
flex usage is causing some layout issues here. Can we use grid?

Reply:
Update: I've refactored the layout from `flex` to `grid` here. Some nice gains in readability, without the sizing/layout issues from the `flex` layout:
https://github.com/hashicorp/react-components/pull/784/commits/8cc180319ce97c319fada6d4e3184c641d068323